tests/test_key_and_signature_datastructures.nim

```diff
@@ -7,49 +7,56 @@
 #
 # at your option. This file may not be copied, modified, or distributed except according to those terms.
 
-import  ../src/eth_keys,
+import  ../src/eth_keys, ../src/private/conversion_bytes,
         ./config
 
 import  unittest
 
 suite "Test key and signature data structure":
-  test "Signing from private key object":
-    for person in [alice, bob, eve]:
-      let
-        pk = initPrivateKey(person.privkey)
-        signature = pk.sign_msg(MSG)
 
-      check:
-        signature.v == person.raw_sig.v
-        signature.r == person.raw_sig.r.u256
-        signature.s == person.raw_sig.s.u256
+  # TODO: For now due to needing hex <-> uint256 conversion
+  # Testing r, s, v direct access is disabled
+  #
+  # test "Signing from private key object":
+  #   for person in [alice, bob, eve]:
+  #     let
+  #       pk = initPrivateKey(person.privkey)
+  #       signature = pk.sign_msg(MSG)
+
+  #     check:
+  #       signature.Fv == person.raw_sig.v
+  #       signature.Fr == hexToByteArrayBE[32](person.raw_sig.r)
+  #       signature.Fs == hexToByteArrayBE[32](person.raw_sig.s)
 
   test "Signing from private key object (ported from official eth-keys)":
     for person in [alice, bob, eve]:
       let
         pk = initPrivateKey(person.privkey)
         signature = pk.sign_msg(MSG)
 
-      check: verify_msg(pk.public_key, MSG, signature)
+      check: verify_msg(pk.Fpublic_key, MSG, signature)
 
-  test "Hash signing from private key object":
-    for person in [alice, bob, eve]:
-      let
-        pk = initPrivateKey(person.privkey)
-        signature = pk.sign_msg(MSG)
+  # TODO: For now due to needing hex <-> uint256 conversion
```

Comment:
Maybe these tests have to be removed completely? If you chose another internal `Signature` representation in the native backed they will fail anyway?